packages/badge/src/react/__stories__/index.story.tsx

```diff
@@ -1,28 +1,37 @@
-import { ValueOf } from '@pluralsight/ps-design-system-util'
-import { storiesOf } from '@storybook/react'
+import { Meta, Story } from '@storybook/react/types-6-0'
 
-import React from 'react'
+import * as React from 'react'
 
-import Badge from '..'
-import * as vars from '../../vars'
+import Badge from '../index'
```

Comment:
is the `../index` related to your ESM work?

Reply:
Yeah. Maybe I should leave until later. 🤷